cmd/gaia/app/app.go

```diff
@@ -97,103 +124,71 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 		tkeyParams:       sdk.NewTransientStoreKey(params.TStoreKey),
 	}
 
+	// init params keeper and subspaces
```

Comment:
Is the param store something we should move into BaseApp?

Reply:
I think that's a reasonable idea for a future PR... this would require further discussion, feel free to open up an issue

Reply:
Great idea! Done: https://github.com/cosmos/cosmos-sdk/issues/4260